scripts/installscripts/buster-install-mopidy3.sh

```diff
@@ -0,0 +1,859 @@
+echo "MPDconfig=\"$MPDconfig\"" >> $PATHDATA/PhonieboxInstall.conf
+
+##################################################### 
+# Folder path for audio files 
+# default: /home/pi/RPi-Jukebox-RFID/shared/audiofolders
+
+clear
+
+echo "#####################################################
+#
+# FOLDER CONTAINING AUDIO FILES
+#
+# The default location for folders containing audio files:
+# /home/pi/RPi-Jukebox-RFID/shared/audiofolders
+#
+# If unsure, keep it like this. If your files are somewhere
+# else, you can specify the folder in the next step.
+# IMPORTANT: the folder will not be created, only the path
+# will be remembered. If you use a custom folder, you must
+# create it.
+"
+
+read -r -p "Do you want to use the default location? [Y/n] " response
+case "$response" in
+    [nN][oO]|[nN])
+    	echo "Please type the absolute path here (no trailing slash)."
+    	echo "Default would be for example:"
+    	echo "/home/pi/RPi-Jukebox-RFID/shared/audiofolders"
+        read INPUT
+        DIRaudioFolders="$INPUT"
+        ;;
+    *)
+    	DIRaudioFolders="/home/pi/RPi-Jukebox-RFID/shared/audiofolders"
+        ;;
+esac
+# append variables to config file
+echo "DIRaudioFolders=\"$DIRaudioFolders\"" >> $PATHDATA/PhonieboxInstall.conf
+echo "Your audio folders live in this dir:"
+echo $DIRaudioFolders
+echo "Hit ENTER to proceed to the next step."
+read INPUT
+
+clear
+
+echo "#####################################################
+#
+# START INSTALLATION
+#
+# Good news: you completed the input.
+# Let the install begin.
+#
+# Get yourself a cup of something. The install takes 
+# between 15 minutes to half an hour, depending on 
+# your Raspberry Pi and Internet connectivity.
+#
+# You will be prompted later to complete the installation.
+"
+
+read -r -p "Do you want to start the installation? [Y/n] " response
+case "$response" in
+    [nN][oO]|[nN])
+    	echo "Exiting the installation."
+    	echo "Your configuration data was saved in this file:"
+    	echo $PATHDATA/PhonieboxInstall.conf
+    	echo
+        exit
+        ;;
+esac
+
+##################################################### 
+# INSTALLATION
+
+# Read install config as written so far
+# (this might look stupid so far, but makes sense once
+# the option to install from config file is introduced.)
+. $PATHDATA/PhonieboxInstall.conf
+
+# power management of wifi: switch off to avoid disconnecting
+sudo iwconfig wlan0 power off
+
+# Install required packages
+sudo apt-get update
+sudo apt-get --yes --allow-downgrades --allow-remove-essential --allow-change-held-packages install libspotify-dev apt-transport-https samba samba-common-bin python-dev python-pip gcc raspberrypi-kernel-headers lighttpd php7.3-common php7.3-cgi php7.3 php7.3-fpm at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero resolvconf spi-tools python-spidev python3-spidev
+
+# use python3.7 as default
+sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1
```

Comment:
Why do you need this? Python 3.7 *is* already the default Python version in Buster.

Reply:
If the user changed it, it was likely for a reason. Maybe they wanted Pythoni 3.8 and you've just undone that and broken something else unncessarily. Personally I think you'd be better of checking the version of Python >= 3.7 (at the start) and erroring out if not.

Reply:
that sounds like a good idea - would you care to create a PR with that change?

Reply:
Sure. I don't use this project so I can't test it directly but I'll do my best when I get some spare moments later.